usrsctplib/user_environment.c

```diff
@@ -85,6 +85,31 @@ read_random(void *buf, size_t size)
 	return;
 }
 
+void
+finish_random(void)
+{
+	return;
+}
+#elif defined(SCTP_USE_OPENSSL_RAND)
```

Comment:
Is there a place where this macro needs to be documented? (e.g. maybe https://github.com/sctplab/usrsctp/blob/master/Manual.md)

Reply:
I think it is used internally by Google. If someone provides a patch for documentation / build systems, I'm willing to merge, but I guess this will not be tested by the buildbots.

Reply:
Taylor, can you add a few lines in the documentation under build systems? (I guess here? https://github.com/sctplab/usrsctp/blob/master/Manual.md#building-the-library-and-the-applications).

Reply:
I feel like documentation for these macros would be out of place there, since it doesn't talk about any build arguments, let alone those with niche usage.

I could add it as an argument in configure.ac/CMakeLists.txt/etc. though; then it would have at least as much documentation as other arguments. What do you think?

Reply:
Yes, that sounds good to me.

Or if you think that is too much for a niche macro like this one, let's add a few lines here to explain it (I just think we should not just use it but at least write something about it).

Reply:
Done.